src/errors/test-run/templates.js

```diff
@@ -108,7 +108,7 @@ export default {
     `),
 
     [TYPE.actionUnsupportedUrlProtocolError]: err => markup(err, `
-        The "${err.argumentName}" argument specifies a URL that uses an unsupported ${err.protocol}:// protocol. Only HTTP and HTTPS are supported, as well as protocol-relative and relative URLs.
+        The "${err.argumentName}" argument specifies a URL that uses an unsupported ${err.protocol}:// protocol. Only HTTP, HTTPS and FILE are supported, as well as protocol-relative and relative URLs.
```

Comment:
There is no such thing as `FILE` protocol. Just use schema in supported protocol enumeration:
`Only relative URLs or absolute URLs with http://, https:// and file:// protocols are supported`. 
What is `protocol-relative URL` though?